examples/embeddable_examples/public/book/book_embeddable.tsx

```diff
@@ -109,7 +109,7 @@ export class BookEmbeddable extends Embeddable<BookEmbeddableInput, BookEmbeddab
 
   getInputAsRefType = async (): Promise<BookByReferenceInput> => {
     const input = this.attributeService.getExplicitInputFromEmbeddable(this);
-    return this.attributeService.getInputAsRefType(input, { showSaveModal: true });
+    return this.attributeService.getInputAsRefType(input, this, { showSaveModal: true });
```

Comment:
Why passing the whole `embeddable` here? From the code it looks it's used only for the title: can it be integrated into the `saveOptions` type?

Reply:
Good point! I've covered this in #76057